FrontEnd/src/App.tsx

```diff
@@ -31,8 +35,14 @@ function App() {
       {userInfo ? <Navbar /> : <p></p>}
       <Routes>
         <Route path="/" element={<Login />} />
-        <Route path="/validate" element={<Validate />} />
-        <Route path="/browse" element={<Browse />} />
+        <Route
+          path="/validate"
+          element={<Validate plot={localPlot} setPlot={setLocalPlot} />}
+        />
+        <Route
+          path="/browse"
+          element={<Browse plot={localPlot} setPlot={setLocalPlot} />}
```

Comment:
I don't think it's necessary/really helps to pass the plot state like this? Ideally what we really want is to have the plot be selected at the beginning as the first step after logging in, and then for the app to remember that, (and for there to be a way to go back and change that). In the meantime, having each page have a dropdown is our temporary solution (and that allows us to use local storage if we want the selected plot to persist across pages).